README.md

```diff
@@ -1 +1,41 @@
 # che-workspace-crd-operator
+
+## Development
+
+### Dependencies
+
+Dependencies in the project are managed by Go Dep. After you added a dependency you need to run the following command to download dependencies to vendor repo and lock file and then commit changes:
+
+```
+dep ensure
+```
+
+Note that dep ensure doesn't automatically change Gopkg.toml which contains dependencies constraints. So, when a dependency is introduced or changed it should be reflected in Gopkg.toml.
+
+### Build docker image
+
+Since it uses Go Dep, you need operator-sdk 0.10.0 where Go Modules is not a default vendor.
+
+To build docker image run the following command in the project's root
+
+```
+operator-sdk build quay.io/che-incubator/che-workspace-crd-controller:7.1.0
+```
+
+### Run controller within K8s/OS cluster
+1. `kubectl apply -f ./deploy/crds`
+2. `kubectl create namespace devworkspace-controller`
+3. For K8s cluster you must to make sure that right domain is set in `./deploy/controller_config.yaml` and `./deploy/registry/local/ingress.yaml`
```

Comment:
Why are you specifying "For K8s cluster" here? This section is about K8s/OS, if this step is for K8s only (not OS) that's ok to be explicit but I think that's for both so I would only mention "Make sure that the right domain is set in....".

Reply:
For the OpenShift case we don't need to provide this information. It is retrieved automatically at start of the controller.

Reply:
Good catch. Actually, I did not try to run it on OpenShift cluster, maybe Route instead of Ingress is needed. For time being I've renamed section to `Run controller within K8s cluster` and then it's needed to test and fill in instructions for OpenShift Cluster